3rdparty/snmalloc/allocator.cpp

```diff
@@ -27,10 +28,18 @@ void* ThreadAllocUntyped::get()
 
 // Enable the Open Enclave PAL(Platform Abstraction Layer) for Open Enclave,
 // see pal_open_enclave.h for details
+#include "./snmalloc/src/override/malloc-extensions.cc"
 #include "./snmalloc/src/override/malloc.cc"
 
+static uint8_t* _heap_start;
```

Comment:
Would it make more sense to store `max_total_heap_size`?  The two variables are leading to confusion for their purpose.

Reply:
Yes, agreed, I will make this change to both allocators.